src/Grid.js

```diff
@@ -11,14 +11,20 @@ export default class Grid {
         const newGrid = new Grid(this.width, this.height);
 
         for (let i = 0; i < this.height; i++) {
-            for (let j = 0; j < this.width  ; j++) {
-                newGrid.grid[i][j].passable = this.grid[i][j].passable;
+            for (let j = 0; j < this.width; j++) {
+                newGrid.grid[i][j].terrain = this.grid[i][j].terrain;
             }
         }
 
         return newGrid;
     }
 
+    copyCells(visibleCells) {
```

Comment:
nit: `visibleCells` doesn't make sense here
e.g. if we were adding better map-making and added a clone tool, we could copy some arbitrary set of cells; nothing about them made them specifically "visible cells" as opposed to... non-visible cells

Reply:
:rabbit2: